project/SparkCassandraConnectorBuild.scala

```diff
@@ -261,11 +350,47 @@ object Dependencies {
 
   val spark = Seq(sparkCore, sparkStreaming, sparkSql, sparkCatalyst, sparkHive, sparkUnsafe)
 
-  val connector = testKit ++ metrics ++ jetty ++ logging ++ akka ++ cassandra ++ spark.map(_ % "provided") ++ Seq(
-    config, guava, jodaC, jodaT, lzf, jsr166e)
+  /**
+    * Dependencies which will be shaded in our distribution artifact and not listed on the
+    * distribution artifact's dependency list.
+    */
+  val shaded = Seq(guava, cassandraDriver)
```

Comment:
I wonder why we list cassandraDriver in shaded here, while in fact it is not really shaded but only included in the jar